lib/tds/binary_utils.ex

```diff
@@ -120,52 +141,56 @@ defmodule Tds.BinaryUtils do
   defmacro freservedbit(), do: quote(do: 0x0 :: size(1))
 
   @doc """
-  A FRESERVEDBYTE is a BYTE value used for padding that does not transmit information. FRESERVEDBYTE fields SHOULD be set to %x00 and MUST be ignored on receipt.
+  A FRESERVEDBYTE is a BYTE value used for padding that does not transmit
+  information.
+
+  FRESERVEDBYTE fields SHOULD be set to %x00 and MUST be ignored
+  on receipt.
   """
   defmacro freservedbyte(), do: quote(do: 0x00 :: size(8))
 
   @doc """
-  A 8-bit signed integer
+  A 8-bit signed integer.
   """
   defmacro int8(), do: quote(do: signed - 8)
 
   @doc """
-  A 16-bit signed integer
+  A 16-bit signed integer.
   """
   defmacro int16(), do: quote(do: signed - 16)
 
   @doc """
-  A 16-bit signed integer
+  A 16-bit signed integer.
```

Comment:
This should be `A 32-bit signed integer.`.
The following documentation is also wrong. 
Would be nice if you could fix this.